src/app/app.module.ts

```diff
@@ -186,6 +189,7 @@ import { MatSliderModule } from '@angular/material/slider';
         ResponsiveMenuModule,
         ScrollingModule,
         SpacerModule,
+        TextFieldModule,
```

Comment:
Is this module actually required?  If you remove it, does the app still build and the new signing page work?

Reply:
I had to add this for the multiline textarea to work as expected

Reply:
I made a change to the message being signed so it displays as a multiline textfield.
It just makes the message a bit more readable when it has line breaks.